modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipFormBase.tsx

```diff
@@ -143,38 +176,127 @@ export function ObjectRelationshipFormBase({
 		});
 	}, [creationLanguageId, objectDefinitions, query]);
 
+	const switchObjects = () => {
+		const previousObjectDefinition1 = {
+			...objectDefinition1,
+		};
+
+		setObjectDefinition1(objectDefinition2);
+
+		setObjectDefinition2(previousObjectDefinition1);
+	};
+	const handleReverseOrder = () => {
+		setValues({
+			objectDefinitionExternalReferenceCode1:
+				objectDefinition2?.externalReferenceCode,
+			objectDefinitionExternalReferenceCode2:
+				objectDefinition1?.externalReferenceCode,
+			objectDefinitionId1: objectDefinition2?.id,
+			objectDefinitionId2: objectDefinition1?.id,
+			objectDefinitionName2: objectDefinition1?.name,
+		});
+
+		switchObjects();
+
+		setReverseOrder(!reverseOrder);
+	};
+
+	const handleHideReverseButton = () => {
```

Comment:
@rebsilva I think this function and the other below could be declared earlier in the file to follow the alphabetical order.